synth.py

```diff
@@ -19,39 +19,56 @@
 from synthtool import gcp
 from synthtool.languages import python
 
-gapic = gcp.GAPICBazel()
+gapic = gcp.GAPICMicrogenerator()
 common = gcp.CommonTemplates()
 versions = ["v1beta1", "v1"]
 
 # ----------------------------------------------------------------------------
 # Generate texttospeech GAPIC layer
 # ----------------------------------------------------------------------------
 for version in versions:
-    library = gapic.py_library(
-        service="texttospeech",
-        version=version,
-        bazel_target=f"//google/cloud/texttospeech/{version}:texttospeech-{version}-py",
-        include_protos=True,
-    )
-    s.move(library / f"google/cloud/texttospeech_{version}")
-    s.move(library / f"tests/unit/gapic/{version}")
-    s.move(library / f"docs/gapic/{version}")
-
-# Use the highest version library to generate import alias.
-s.move(library / "google/cloud/texttospeech.py")
+    library = gapic.py_library(service="texttospeech", version=version,)
+    s.move(library, excludes=["setup.py", "docs/index.rst"])
 
 # Fix bad docstrings.
 s.replace("**/gapic/*_client.py", r'\\"(.+?)-\*\\"', r'"\1-\\*"')
 
+# Sphinx interprets `*` as emphasis
+s.replace(
+    ["google/cloud/**/client.py", "google/cloud/**/cloud_tts.py"],
+    "((en)|(no)|(nb)(cmn)|(yue))-\*",
+    "\g<1>-\*",
```

Comment:
I'm not sure I understand this part. Is this indicative of an error in the template, bad escaping on the part of the generator, or just garbage input and garbage sphinx?

Reply:
Sphinx/RST interprets the `*` as emphasis. I'd expect the generator to escape it? The proto comments look valid to me

Source protos:
https://github.com/googleapis/googleapis/blob/b8c7c4e8da56547fc32bee9b563d306fadabdb6f/google/cloud/texttospeech/v1/cloud_tts.proto#L59-L66